### PR TITLE
It strategy corrections

### PR DESCRIPTION
**Pull request by remyb2canada**
Two corrections:

1) Changed wording in the section "Program Statement"
Original wording made it sound like the problem is that we have too many applications in our portfolio. It wasn't the intent. The intent was to raise awareness on the increasing complexity technology (and departments) are facing. 

2) Corrected display of numbered list under Short Term Outcomes

### CODE REVIEW

**Comment by jaysonmc** [comment]
@remyb2canada 
Would be a good opportunity to add a link to the PM PoC as guidance as well, if we have a link and info for that?

**Comment by remyb2canada** [comment]
> @remyb2canada
> Would be a good opportunity to add a link to the PM PoC as guidance as well, if we have a link and info for that?

@jaysonmc that's for another document. Not that one.

**Comment by jaysonmc** [comment]
woops. Got my document titles mixed up.

File: _pages/en/esdc-it-strategy.md [hunk 1]
```diff
@@ -28,11 +28,14 @@ In addition, ESDC has started investing in a [Business Delivery Modernization](h
 
 ## Problem Statement
 
-As explained in [ESDC's Medium-Term Planning](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Briefings/MTP-Initiative-Mapping_2018.docx), ESDC operates a portfolio of approximately 500 applications, 101 of which are mission critical.
+Moving to digital is creating a world that is highly complex.
```
Comment (gabrielcossette) [inline _pages/en/esdc-it-strategy.md]: I see where you are getting at and I agree... but we need to work on simplifying the complexity as much as possible. Loosely coupled parts, clear documented APIs, automated tests... We could reword "highly" to something less strong, at your discretion.

**Pull request merged**